sources/packages/web/src/components/generic/StatusChip.vue

```diff
@@ -24,42 +20,131 @@ export default {
       type: String,
       required: false,
     },
+    /**
+     * When property is true its is a normal status like chip
```

Comment:
I am not following this part. The `StatusChip` is supposed to be generic. Why are there different behaviors for designation or restriction?

Reply:
![image](https://user-images.githubusercontent.com/77353155/182662511-cd8393a9-dc89-4a30-a0bf-a3ad40018736.png)

Reply:
first 2 are for designation or restriction